### DISCUSSION TITLE
How to create a custom script/cli command

### DISCUSSION THREAD

**Discussion by dretsa** [keystonejs/keystone#8979] [Questions]
I want to fetch some data from an external API to display neatly using the UI of Keystone.

I can not find any guides/docs about this. Is this even possible?

I ideally want to run the script multiple times a day via a cronjob. Can I do this via Keystone? It would be nice to take advantage of the Prisma ORM, without having to set up the entities/DB connection again for a separate script/app.

**Comment by unrevised6419** [comment]
Hey, @colorninja! I think this example may help you [examples/script](https://github.com/keystonejs/keystone/tree/main/examples/script). You can use it in combination maybe with [`cron`](https://github.com/kelektiv/node-cron)
